etna/ciim/client.py

```diff
@@ -405,14 +405,15 @@ def fetch_all(
 
         Used to fetch a all items by for the given identifier(s).
 
-        Fetch all metadata with a generic identifier, iaid or replicaId (rid).
+        Fetch all metadata with a generic identifier, IAID's of various formats or
+        replicaId (rid).
 
         Keyword arguments:
 
         ids:
             Generic identifiers. Matches on references_number or iaid
-        iaids:
-            Return matches on Information Asset Identifier
+        metadata_ids:
+            Return match on @admin.id for various Information Asset Identifier (IAID) formats.
```

Comment:
Again, the `ids` option should really be updated to support `metadata_id` values, and we should do away with this option really.

Reply:
Happy to approve the API client changes if that's out our control for now.